### PR TITLE
Fix setup script

### PR DESCRIPTION
**Pull request by skatsaounis**
Includes the following:

- Aligns option `sn|--skip-lxn` with help text of the script
- Correctly uses `kvm_network_prefix` for the second interface rather than `control_network_prefix`
- Adds a missing dollar to a variable which was causing an error during setup
- Replaces `${MAAS_MANAGEMENT_IP_RANGE}/24` with `${kvm_network_prefix}.0/24` to properly select the subnet
- Uncomment part that enables HTTPS endpoint of host LXD daemon
- Fixes typos and removes extra spaces

### CODE REVIEW

**Review by SK1Y101 — APPROVED** [review]
LGTM

File: setup-region-via-ssh.bash [hunk 1]
```diff
@@ -30,71 +30,71 @@ network:
...
-echo 
+echo
 echo "##########################"
 echo "Creating a MAAS admin user"
 sudo maas createadmin --username maas --password maas --email maas@example.com
 
-echo 
+echo
 echo "###########################"
 echo "Logging using admin profile"
```
Comment (SK1Y101) [inline setup-region-via-ssh.bash:77]: Logging *in* using admin profile

File: setup-region-via-ssh.bash [hunk 2]
```diff
@@ -30,71 +30,71 @@ network:
...
 
-echo 
+echo
 echo "###############################"
 echo "Starting DHCP on second network"
 # target_network=$(maas admin subnets read | jq .[].name --raw-output | grep "10\.20\.0\.")
 #
 rack_controllers=$(maas admin rack-controllers read)
 target_rack_controller=$(echo $rack_controllers | jq --raw-output .[].system_id)
 target_fabric_id=$(echo $rack_controllers | jq '.[].interface_set[].links[] | select(.subnet.name | startswith("10.20.0.")) | .subnet.vlan.fabric_id')
```
Comment (SK1Y101) [inline setup-region-via-ssh.bash:87]: `10.20.0.` is assuming the user has not changed the config option, should be changed to `${control_network_prefix}`

**Review by SK1Y101 — COMMENTED** [review]
just re-setup my own dev instance using these changes, small issue was noted as commented, with two extra small changes

File: setup-region-via-ssh.bash (hunk 2 above)
Comment (skatsaounis) [reply setup-region-via-ssh.bash]: done

File: setup-region-via-ssh.bash (hunk 1 above)
Comment (skatsaounis) [reply setup-region-via-ssh.bash]: I finally used `Login using admin profile` as a better alternative.Thank you for pointing it

File: setup-region-via-ssh.bash [hunk 3]
```diff
@@ -30,71 +30,71 @@ network:
...
+echo
 echo "##########################"
 echo "Creating a MAAS admin user"
 sudo maas createadmin --username maas --password maas --email maas@example.com
 
-echo 
+echo
 echo "###########################"
 echo "Logging using admin profile"
 maas login admin "http://${container_ip}:5240/MAAS/api/2.0/" $(sudo maas apikey --username=maas)
```
[the remark this answers is not in the document]
Comment (tmerten) [reply setup-region-via-ssh.bash:78]: Yeah, if I'm not mistaken the `/MAAS` URL should redirect.

**Review by tmerten — APPROVED** [review]
Looks good to me as well, I think we can land that and see if we need to / can change the url later.

**Pull request merged**